### PR TITLE
Fix errors causing fiat deposits & withdraw to be broken

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: app/controllers/private/fund_sources_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module Private
   class FundSourcesController < BaseController
+    skip_before_action :verify_authenticity_token
```
Comment (yivo) [inline app/controllers/private/fund_sources_controller.rb]: @vpetrusenko LOL. Disabling security is not a fix.

**Review by yivo — CHANGES_REQUESTED** [review]

File: app/controllers/private/fund_sources_controller.rb (hunk 1 above)
Comment (vpetrusenko) [reply app/controllers/private/fund_sources_controller.rb]: ...my bad, sorry

**Comment by yivo** [comment]
@vpetrusenko Please, continue @dkkoval's work at https://github.com/rubykube/peatio/pull/289.

**Comment by vpetrusenko** [comment]
@yivo pls check if i understand and do everything correctly

**Comment by yivo** [comment]
Connected issues: #276, #290.

File: app/assets/javascripts/funds.js.coffee [hunk 2]
```diff
@@ -51,4 +51,11 @@ $(document).on 'click', '[data-clipboard-text], [data-clipboard-target]', (e) ->
   # Emulate click.
   $action.click()
 
+  app.config ['$httpProvider', ($httpProvider) ->
```
Comment (yivo) [inline app/assets/javascripts/funds.js.coffee]: @vpetrusenko Sorry, but what this is doing here? Did you ever check this? app is undefined. Why do you place this into document ready?

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by vpetrusenko** [comment]
One more connected issue: #130

**Comment by yivo** [comment]
@vpetrusenko doesn't work

screen shot 2018-01-15 at 1 08 22 pm
screen shot 2018-01-15 at 1 08 08 pm

**Comment by vpetrusenko** [comment]
2018-01-15-141201_1680x1050_scrot
2018-01-15-141148_1680x1050_scrot

**Comment by vpetrusenko** [comment]
@yivo i think it works correctly now

File: app/controllers/concerns/withdraws/withdrawable.rb [hunk 3]
```diff
@@ -32,7 +39,7 @@ def fetch
     def withdraw_params
       params[:withdraw][:currency] = channel.currency
       params[:withdraw][:member_id] = current_user.id
-      params.require(:withdraw).permit(:fund_source_id, :member_id, :currency, :sum)
+      params.require(:withdraw).permit(:member_id, :currency, :sum)
```
Comment (mod) [inline app/controllers/concerns/withdraws/withdrawable.rb]: please verify if the form is posting fund_source_id and allow it,
I guess when you select your wallet JS should set the id before we post the request.

**Review by mod — CHANGES_REQUESTED** [review]

**Comment by vpetrusenko** [comment]
2018-01-16-093958_1680x1050_scrot
2018-01-16-094135_1680x1050_scrot

**Comment by vpetrusenko** [comment]
so it works with btc

**Review by yivo — APPROVED** [review]

**Comment by vpetrusenko** [comment]
Following issues should be closed: #285 #290 #276 #130

**Review by mod — APPROVED** [review]

**Comment by mkuvsh** [comment]
Reviewed and approved

**Pull request merged**